docs/usage-tui.md

```diff
@@ -75,7 +75,7 @@ Ariadne using the 12 word mnemonic.
 When you select a wallet or an account, you can see its balance and send 
 transactions using the form provided. You must fill at least one receiver's 
 address. To add or remove address lines, use `[ - ]` / `[ + ]` buttons on
-the right. If the wallet has a passphrase, you need to specify that as well. 
+the right. You will be asked for a passphrase, even if you chose an empty one. 
```

Comment:
Wait a second, will it ask for a password, when I use the send form (which already has a password input)?

Reply:
Nope, it will only ask if you put the command in the `Repl` (in TUI).

Reply:
Oh no, I just checked, it does ask you for one anyway, will fix this.

Reply:
I was asking myself why I removed the password from `UiSendArgs` if there was a field in the form and then I remembered: there is no field anymore.

You can see it was removed some time ago, in the second commit, I suppose it was discussed, but I do not remember. Should I put it back in?

Reply:
Well, I think that existing forms with password fields should work as before.
That is, they should preemptively provide a password to the password manager.
The password should be explicitly asked only when you type a command in REPL.